### PR TITLE
No need to remove MongoDB log files

### PR DESCRIPTION
**Pull request by sean797**
ping @chris1984

### CODE REVIEW

**Comment by chris1984** [comment]
@sean797 Thanks for catching that.

File: hooks/post/30-upgrade.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def remove_legacy_mongo
   if `rpm -q mongodb --queryformat=%{version}`.start_with?('2.')
     logger.warn 'removing MongoDB 2.x packages, config and log files.'
     Kafo::Helpers.execute('yum remove -y mongodb-2* mongodb-server-2* > /dev/null 2>&1')
-    Kafo::Helpers.execute('rm -rf /etc/mongod.conf /var/log/mongodb')
+    Kafo::Helpers.execute('rm -rf /etc/mongod.conf')
```
Comment (ekohl) [inline hooks/post/30-upgrade.rb]: Looking at this, why do we use `-r` on a single file? That's just asking for errors. We could also use pure ruby with `File.unlink('/etc/mongod.conf') if File.exist?('/etc/mongod.conf')`

Comment (chris1984) [reply hooks/post/30-upgrade.rb]: Good catch, sorry for merging so fast.

Comment (chris1984) [reply hooks/post/30-upgrade.rb]: I will make a new PR to fix it

**Pull request merged**